src/pydocstyle/checker.py

```diff
@@ -390,6 +411,174 @@ def check_starts_with_this(self, function, docstring):
             if first_word.lower() == 'this':
                 return violations.D404()
 
+    @staticmethod
+    def _get_leading_words(line):
+        """Return any leading set of words from `line`.
+
+        For example, if `line` is "  Hello world!!!", returns "Hello world".
+        """
+        result = re("[A-Za-z ]+").match(line.strip())
```

Comment:
The numpy convention only allows the section names specified above, no accents there. I could change the method's name or documentation if it will be clearer.